chain/emitter/app.go

```diff
@@ -202,22 +213,29 @@ func (app *App) DeliverTx(req abci.RequestDeliverTx) abci.ResponseDeliverTx {
 			"extra": extra,
 		})
 	}
+
+	accsInTx := []string{}
+	accMap := make(map[string]bool)
```

Comment:
Why we need to declare this map, we can't use app.AddAccountsInBlock(app.accsInTx...)?